jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -98,16 +100,30 @@ private Tracer(
     this.version = loadVersion();
 
     tags.put(Constants.JAEGER_CLIENT_VERSION_TAG_KEY, this.version);
-    String hostname = getHostName();
-    if (hostname != null) {
-      tags.put(Constants.TRACER_HOSTNAME_TAG_KEY, hostname);
+    Object hostNameTag = tags.get(Constants.TRACER_HOSTNAME_TAG_KEY);
```

Comment:
use `if (tags.containsKey(Constants.TRACER_HOSTNAME_TAG_KEY))` to avoid leaking variable name to outer scope

Reply:
That’s me being consistent with the code below. I will modify to your tastes.